### DISCUSSION TITLE
Ground Items: Loot price short-hand option (Easy Feature Add)

### DISCUSSION THREAD

**Discussion by matthewmachart** [runelite/runelite#16824] [Ideas]
I like being able to see both the GE and HA prices on drops while doing slayer, but after a few enemies the screen is **VERY** cluttered with text.

After playing a while, we all know the first number is GE, and the second number is HA. 

An option to short-hand this to **just numbers** would be a fantastic option to reduce the screen clutter.

![image](https://github.com/runelite/runelite/assets/95005441/31611232-7687-458e-81d7-6745e88d0ab9)

**Comment by geheur** [comment]
If you do this you may as well remove the parentheses. And make it a different color to further differentiate it from the stack quantity, just like the despawn timer.

`Coins (132) - 132` for example (no need for ha value, that's not very useful imo).

Personally though I just don't show either the ge or ha value. ha mostly tracks ge value and ge value I can identify from the price tier colors. You can also change the color of the ground item text from white to gray to make it less contrasting with the floor.

**Reply by matthewmachart** [reply]
I thought about colors and no parentheses, but it would conflict to much with the colored price brackets.

As for price tier colors... on a lot of slayer task, most creatures aren't dropping anything above the color white until higher levels... but you still wanna pick up some stuff to make money while doing it.

Something like: 
**Gold bar (96/180)** 
would be an even tighter way to short-hand the prices
